PowershellTools.Common/ServiceManagement/PowershellHostServiceExceptionDetails.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PowerShellTools.Common.ServiceManagement
+{
+    [DataContract]
+    class PowershellHostServiceExceptionDetails
+    {
+        public static readonly PowershellHostServiceExceptionDetails Default = new PowershellHostServiceExceptionDetails();
+
+        [DataMember]
+        public String Message { get; private set; }
+
+        public PowershellHostServiceExceptionDetails()
+        {
+            this.Message = "There is a problem in the powershell host service.";

```

Comment:
If this exception message can ever be seen by a user, we should make localizable